pkg/services/grpcserver/interceptors/auth.go

```diff
@@ -101,6 +115,14 @@ func (a *Authenticator) getSignedInUser(ctx context.Context, token string) (*use
 		return nil, fmt.Errorf("service account is disabled")
 	}
 
+	if signedInUser.Permissions[signedInUser.OrgID] == nil {
```

Comment:
we could also add a `if signedInUser == nil` - not sure if this edge case is possible (deleted user with stale api key?)

Reply:
added a check in [ebd85d4](https://github.com/grafana/grafana/pull/56059/commits/ebd85d41fbaa044ba865a27b230e2f3948948ca6)